staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/generic/doc.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This package contains a typed wrapper over cache SharedIndexInformer
+// and Lister (maybe eventually should have a home there?)
```

Comment:
I am worried that people will reuse this where it is because it looks useful. I think eventually we should move it there, yes. Until then, what do you think about putting this in an internal package to make it hard for other people using it to make it hard for you to move it eventually?